### PR TITLE
Include sys.argv in error emails

### PR DESCRIPTION
**Pull request by neilisaac**
Sometimes my team shares base tasks. When we get an error email, it's sometimes unclear which upstream task requires() the one that failed.

Following @Tarrasch's suggestion of adding command line arguments to address this.

### CODE REVIEW

**Comment by neilisaac** [comment]
Example email body:

```
A task failed when running. Most likely run() raised an exception.

Name: SubTask

Parameters:
  p: some option

Command line:
  'testtask.py' '--local-scheduler' 'TestTask' '--p' 'asdf'

Runtime error:
Traceback (most recent call last):
  File "/home/nisaac/code/luigi/luigi/worker.py", line 172, in run
    new_deps = self._run_get_new_deps()
  File "/home/nisaac/code/luigi/luigi/worker.py", line 114, in _run_get_new_deps
    task_gen = self.task.run()
  File "testtask.py", line 9, in run
    raise Exception('Intentional failure')
Exception: Intentional failure
```

File: luigi/worker.py [hunk 1]
```diff
@@ -509,7 +510,8 @@ def _email_task_failure(self, task, formatted_traceback):
 
     def _email_error(self, task, formatted_traceback, subject, headline):
         formatted_subject = subject.format(task=task, host=self.host)
-        message = notifications.format_task_error(headline, task, formatted_traceback)
+        command = ' '.join("'{}'".format(arg) for arg in sys.argv)
```
Comment (Tarrasch) [inline luigi/worker.py]: This makes the command look weird and hard to read I think. What about simply

```
command = subprocess.list2cmdline(sys.argv)
```

**Comment by Tarrasch** [comment]
LGTM. But the formatting looks strange to me. See my inline comment suggestion. :)

**Comment by Tarrasch** [comment]
LGTM! Will merge on green :)

**Pull request merged**